hastexo/tasks.py

```diff
@@ -266,7 +266,7 @@ def run(self, **kwargs):
                 'error_msg': textwrap.shorten(e.error_msg, width=256),
                 'ip': None,
                 'user': "",
-                'key': "",
+                'key': self.stack_key if e.suspend else '',
```

Comment:
This looks good to me, but I think Future Maari (and anyone else reviewing this bit of code in the future) will appreciate a brief explanation _why_ this logic applies here. :) Perhaps add a comment to that effect right above the `stack_data` definition in line 264?

Reply:
Fair point. And on second thought, I was thinking that perhaps it's better to not touch the key at all, in case of a failed resume. So instead of overwriting it with the same key that's already there, remove it from the fields to be updated.